pyproject.toml

```
@@ -34,9 +34,9 @@ dependencies = [
     "pydantic>=2.3, <3",
     "scalecodec==1.2.11",
     "uvicorn",
-    "bittensor-drand>=0.5.0",
-    "bittensor-wallet>=3.1.0",
-    "async-substrate-interface>=1.3.1"
+    "bittensor-drand>=1.0.0",
```

let's bound this above by the next major
```suggestion
    "bittensor-drand~=1.0.0",
```